pyodide-build/pyodide_build/tests/test_cli.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from typer.testing import CliRunner  # type: ignore[import]
+
+from pyodide_build.cli import new_package
+
+runner = CliRunner()
+
+
+@pytest.fixture
+def packages_temp_path(tmp_path):
+    yield tmp_path
```

Comment:
Can't we just use tmp_path then?

Reply:
Good point. I think I added that to use the same tmp_path for multiple tests but I only write a single test so we don't need that fixture now.